fakes/image.go

```diff
@@ -291,6 +291,14 @@ func (i *Image) Found() bool {
 	return !i.deleted
 }
 
+func (i *Image) AnnotateRefName(refName string) error {
+	return nil
+}
+
+func (i *Image) GetAnnotateRefName() (string, error) {
+	return "", nil
+}
+
```

Comment:
Should we be updating the fake somehow? To avoid confusing future consumers of this fake...

Reply:
Well, I added an implementation for those methods, the annotation is being added during `Save()` (if it was provided) and a test method `Annotations()` was exposed, that will help users of this implementation to test some behaviors.